odoo/service/security.py

```diff
@@ -18,7 +18,8 @@ def compute_session_token(session, env):
 
 def check_session(session, env):
     self = env['res.users'].browse(session.uid)
-    if odoo.tools.misc.consteq(self._compute_session_token(session.sid), session.session_token):
+    computed_session_token = self._compute_session_token(session.sid)
```

Comment:
To make it more concise/readable, could we rename this variable to `expected` or something similar?

Reply:
expected_token would have been better.